pkgs/applications/networking/mailreaders/thunderbird/packages.nix

```
@@ -16,6 +16,40 @@ rec {
       ./no-buildconfig.patch
     ];
 
+    meta = with lib; {
+      description = "A full-featured e-mail client";
+      homepage = "https://thunderbird.net/";
+      maintainers = with maintainers; [ eelco lovesegfault pierron vcunat ];
+      platforms = platforms.unix;
+      badPlatforms = platforms.darwin;
+      broken = stdenv.buildPlatform.is32bit; # since Firefox 60, build on 32-bit platforms fails with "out of memory".
+                                             # not in `badPlatforms` because cross-compilation on 64-bit machine might work.
+      license = licenses.mpl20;
+    };
+    updateScript = callPackage ./update.nix {
+      attrPath = "thunderbird-91-unwrapped";
+    };
+  }).override {
+    geolocationSupport = false;
+    webrtcSupport = false;
+
+    pgoSupport = false; # console.warn: feeds: "downloadFeed: network connection unavailable"
+  };
+  thunderbird = (buildMozillaMach rec {
```

```suggestion
  thunderbird-102 = (buildMozillaMach rec {
```

So that it can be selected by version as well. The primary attribute `thunderbird` can then be an alias that we move when needed.

---

Shall the same be done for thunderbird-bin?